File: app/src/main/java/com/devx/raju/di/module/ActivityModule.java
package com.devx.raju.di.module;

import com.devx.raju.ui.activity.GithubListActivity;

import dagger.Module;
import dagger.android.ContributesAndroidInjector;

@Module
public abstract class ActivityModule {

    @ContributesAndroidInjector()
    abstract GithubListActivity contributeGithubListActivity();
}

File: app/src/main/java/com/devx/raju/di/module/ActivityModule.kt
package com.devx.raju.di.module

import com.devx.raju.ui.activity.GithubListActivity
import dagger.Module
import dagger.android.ContributesAndroidInjector

@Module
abstract class ActivityModule {
    @ContributesAndroidInjector
    abstract fun contributeGithubListActivity(): GithubListActivity?
}